spark/src/main/scala/geotrellis/spark/io/s3/S3AttributeStore.scala

```diff
@@ -33,6 +33,9 @@ class S3AttributeStore(bucket: String, rootPath: String) extends AttributeStore[
 
   def path(parts: String*) = parts.filter(_.nonEmpty).mkString("/")
 
+  def attributePath(id: LayerId): String =
+    path(rootPath, "_attributes", s"*__${id.name}__${id.zoom}.json")
```

Comment:
I am pretty certain that `DeleteObjectRequest` to s3client does not glob. Which presents a really unfortunate problem here. The reason attribute name is listed first here is so we can use `listObjects` to implement `.readAll(attributeName)`. Now we need to do the opposite, get a list of all attributes per layer.

This would not be a problem if we stored attributes with each layer... which we switched from.